src/Sylius/Component/Core/Model/Taxon.php

```diff
@@ -26,48 +26,62 @@ class Taxon extends BaseTaxon implements TaxonInterface
     protected $products;
 
     /**
-     * @var ImageInterface
+     * @var ArrayCollection
      */
-    protected $image;
+    protected $images;
```

Comment:
Shouldn't be `Collection|TaxonImageInterface[]` if `getImages()` returns `Collection|TaxonImageInterface[]`?